### DISCUSSION TITLE
Units speed and range

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#139] [Ideas]
To check if the position/time of a unit (or a single vehicle/ship/aircraft) is correct, user could set a maximum and cruise speed.
So, if a second position in space and time is assigned, system could check if it is "too far" from the preceding position, or at least tell the user: "If you place here the marker, this (e.g.) ship should have had an average speed of (e.g.) 200 knots". So you can immediately inform if the position is possible or impossible. The same could be done for the unit range without refuelling.
Also an infantry unit has a speed (very low but measurable), so this can apply to everything of the map.
What do you think about it?
Thank you
Federico

**Comment by kjellmf** [comment]
Great idea!
